### PR TITLE
Merge Branch2.0 into master

### PR DESCRIPTION
**Pull request by mariusvniekerk**
We've been running on branch2.0 for around half a year now.  Time to make this master

Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

Fixes https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/34

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
We might want to wait for #39 to land as well

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +40,8 @@ install:
       echo "Installing a fresh version of Miniconda."
       MINICONDA_URL="https://repo.continuum.io/miniconda"
       MINICONDA_FILE="Miniconda3-latest-MacOSX-x86_64.sh"
+      # temporarily use an older miniconda so that we can get a build out
+      MINICONDA_FILE="Miniconda3-4.5.4-MacOSX-x86_64.sh"
```
Comment (nsoranzo) [inline .travis.yml]: This shouldn't be needed any more now that  osx-64/conda-forge-ci-setup-2.1.1-py37_0.tar.bz2 has been published on anaconda.org .

Comment (mariusvniekerk) [reply .travis.yml]: we'll change it in 2.1.2.  Mostly harmless for now

Comment (mariusvniekerk) [reply .travis.yml]: and this branch can't get any more commits really in any case since they all make releases

Comment (nsoranzo) [reply .travis.yml]: Sure, I didn't notice this was open directly from the `branch2.0` branch.

**Comment by jakirkham** [comment]
We can just push this to both branches for now. Though I agree we should switch back to `master` in conda-smithy.

**Comment by mariusvniekerk** [comment]
Yeah I don't intend to delete this branch for a while

**Comment by mariusvniekerk** [comment]
We may want to make a 1.x branch just for incase

**Comment by jakirkham** [comment]
That sounds fine. Though any feedstock relying on the 1.x version likely doesn't have it pinned as it is likely quite old.

**Pull request merged**